CHANGELOG.asciidoc

```diff
@@ -6,6 +6,11 @@
 ////////////////////////////////////////////////////////////
 // Template, add newest changes here
 
+==== Added
+
+*MetricBeat*
+- Add postgresql statement metricset. {issue}7048[7048] {pull}7060[7060]
```

Comment:
Add it to the existing "Added" section for version HEAD please :slightly_smiling_face: